server/cloud_tag.go

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package server
+
+import (
+	"bytes"
+	"context"
+	"net/http"
+	"strings"
+	"text/template"
+	"time"
+
+	"github.com/google/go-github/v39/github"
+	"github.com/mattermost/mattermost-mattermod/model"
+	"github.com/mattermost/mattermost-server/v6/shared/mlog"
+)
+
+const (
+	cloudTagName = "tags/cloud"
+	dryRunFlag   = "--dry-run"
+	forceFlag    = "--force"
+)
+
+type cloudTagResult struct {
+	Tagged  map[string]string
+	Skipped []string
+	DryRun  bool
+}
+
+// HasCloudTag is true if body contains "/cloud-tag"
+func (e *issueCommentEvent) HasCloudTag() bool {
+	return strings.Contains(strings.TrimSpace(e.Comment.GetBody()), "/cloud-tag")
+}
+
+func (s *Server) createCloudTag(ctx context.Context, issue *model.Issue, comment, user string) (*cloudTagResult, error) {
+	if issue.State == model.StateClosed || issue.RepoName != processRepo { // we will perform this only on mm-server issues
+		return nil, nil
+	}
+
+	// Don't start process if the user is not a core committer
+	if !s.IsOrgMember(user) {
+		return nil, nil
+	}
+
+	// We are using a mutex for multi-deployment setups
+	mx := s.Store.Mutex()
+	if err := mx.Lock(ctx); err != nil {
+		return nil, err
+	}
+	defer func() {
+		if err := mx.Unlock(); err != nil {
+			mlog.Error("unable to release lock", mlog.Err(err))
+		}
+	}()
+
+	result := &cloudTagResult{
+		Tagged: map[string]string{},
+	}
+
+	// Check if the args are correct
+	command := getCloudTagCommand(comment)
+	args := strings.Split(command, " ")
+	mlog.Info("Args", mlog.String("Args", comment))
+	var force, dryRun bool
+	if len(args) >= 2 {
+		force = args[1] == forceFlag
+		dryRun = args[1] == dryRunFlag
+		result.DryRun = dryRun
+	}
+
+	tagName := cloudTagName + "-" + time.Now().Format("2006-01-02") + "-1"
```

Comment:
Curious, what is the logic behind the "-1". @phoinixgrr - do you know?

Reply:
🤷

Reply:
Is it okay if we remove it?

Reply:
I really have no idea. 
If I had I would had provided some context in the previous question

Reply:
@spirosoik - is it okay if we remove the "-1" at the end of the tag?

Reply:
ping @spirosoik